xbmc/windowing/X11/WinSystemX11GLContext.cpp

```diff
@@ -268,7 +268,7 @@ bool CWinSystemX11GLContext::RefreshGLContext(bool force)
   std::transform(gpuvendor.begin(), gpuvendor.end(), gpuvendor.begin(), ::tolower);
   bool isNvidia = (gpuvendor.compare(0, 6, "nvidia") == 0);
   bool isIntel = (gpuvendor.compare(0, 5, "intel") == 0);
-  std::string gli = (getenv("GL_INTERFACE") != nullptr) ? getenv("GL_INTERFACE") : "";
+  std::string gli = (getenv("KODIGL_INTERFACE") != nullptr) ? getenv("KODIGL_INTERFACE") : "";
```

Comment:
I would rename all environment variables to KODI_, not KODIXX_

Why don't to search for all? It won't take you more than just a couple of minutes. If you don't do it now, it won't be done ever